### DISCUSSION TITLE
Please discontinue the Torrent feature

### DISCUSSION THREAD

**Discussion by PCUser2021** [setvisible/DownZemAll#189] [Ideas]
I'm sorry, I'm not going to offend, but the Torrent feature in download managers is not a good feature at all, why? Because all download managers have their Torrent feature is incomplete and slow and do not offer the features that other Torrent clients offer. Whoever wants to download Torrent goes to qBittorrent or if she/he is very unaware, he goes to uTorrent. In my opinion, adding the Torrent feature only wastes the developer's time to download managers, and also only makes the program heavier. As I said, many Torrent clients have been developed that are both much better and more professional and very well developed, and I did not even see the seeders from FDM or DownZemAll! when I download the Torrent!.
I think you should focus on Chrome Extention which has bugs and does not work properly.

Regards
A DownThemAll user from 2004 to 2016!

**Comment by setvisible** [comment]
Hi
Personally I like the open-source torrent feature very much, I don't want it disappear.
Regards
